tests/status.test.pact.js

```diff
@@ -0,0 +1,44 @@
+const { Pact } = require('@pact-foundation/pact');
+const api = require('./index');
+
+describe('The API', () => {
+  let url = 'http://localhost:3000';
+
+  // Copy this block once per interaction under test
+  describe('Token Status', () => {
+    beforeEach(() => {
+      const interaction = {
+        uponReceiving: 'should verify token is correct and return true or false',
+        withRequest: {
+          method:  'GET',
+          path: '/auth',
+          query:  'token=asdfjkl',
```

Comment:
This should be passed in an `Authorization: Bearer ${token}` header not a query string.